client/client.go

```diff
@@ -377,9 +425,9 @@ func (c *Config) Debugf(message string, format ...any) {
 	}
 }
 
-// func (c *Config) SetAlertWebhook(f webhook) {
-// 	c.AlertWebhook = f
-// }
+func (c *Config) SetAlertWebhook(f webhook, block bool) {
```

Comment:
We are not using `block` parameter here. It can be removed, I guess.

Reply:
oh shoot! I forgot to get rid of that